clients/privacy-center/app/server-environment.ts

```diff
@@ -287,156 +255,99 @@ export const loadStylesFromFile = async (
  * Loads all the ENV variable settings, configuration files, etc. to initialize the environment
  */
 // eslint-disable-next-line no-underscore-dangle,@typescript-eslint/naming-convention
-export const loadPrivacyCenterEnvironment =
-  async (): Promise<PrivacyCenterEnvironment> => {
-    if (typeof window !== "undefined") {
-      throw new Error(
-        "Unexpected error, cannot load server environment from client code!"
-      );
-    }
-    // DEFER: Log a version number here (see https://github.com/ethyca/fides/issues/3171)
-    if (process.env.NODE_ENV === "development") {
-      console.log("Load Privacy Center environment for session...");
-    }
+export const loadPrivacyCenterEnvironment = async ({
+  customPropertyPath,
+}: {
+  customPropertyPath?: string;
+}): Promise<PrivacyCenterEnvironment> => {
```

Comment:
can we update the interface here to have the prop be optional instead of requiring it and forcing the use of an empty `{}` in existing code?
```suggestion
export const loadPrivacyCenterEnvironment = async ({
  customPropertyPath,
}: {
  customPropertyPath?: string;
} | undefined): Promise<PrivacyCenterEnvironment> => {
```

Reply:
Yess, that would be better. Let me change that.